independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/apiDefinition.swagger.json

```diff
@@ -0,0 +1,201 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Default title",
+    "description": "This is a connector to the Open Trivia Database to allows access to it's fun trivia questions",
+    "version": "1.0",
+     "contact": {
+      "name": "Kiveshan Naidoo, Rishay Sunder",
+      "email": "kiveshannaidoo9@gmail.com"
+    }
+  },
+  "host": "opentdb.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/api_category.php": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Get Categories",
+        "description": "Get the categories along with their IDs'",
+        "operationId": "GetCategories",
+        "parameters": []
+      }
+    },
+    "/api.php": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Get Question",
+        "description": "Gets a certain amount of questions",
+        "operationId": "GetQuestion",
+        "parameters": [
+          {
+            "name": "amount",
+            "in": "query",
+            "required": true,
+            "type": "integer",
+            "description": "Amount",
+            "x-ms-summary": "Amount of questions"
+          },
+          {
+            "name": "category",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "description": "Category_ID",
+            "x-ms-summary": "Category_ID",
+            "enum": [
+              9,
+              10,
+              11,
+              12,
+              13,
+              14,
+              15,
+              16,
+              17,
+              18,
+              19,
+              20,
+              21,
+              22,
+              23,
+              24,
+              25,
+              26,
+              27,
+              28,
+              29,
+              30,
+              31,
+              32
+            ]
+          },
+          {
+            "name": "difficulty",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Difficulty",
+            "x-ms-summary": "Difficulty",
+            "enum": [
+              "easy",
+              "medium",
+              "hard"
+            ]
+          },
+          {
+            "name": "type",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "enum": [
+              "boolean",
+              "multiple"
+            ],
+            "description": "Type",
+            "x-ms-summary": "Type of question"
+          }
+        ]
+      }
+    },
+    "/api_count.php": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}

```

Comment:
@Kiveshan empty Schema is not allowed, please define a proper schema or you can delete the empty schema.